x-pack/plugins/apm/public/components/app/mobile/error_group_details/index.tsx

```diff
@@ -0,0 +1,251 @@
+      kuery,
+      rangeFrom,
+      rangeTo,
+      serviceGroup,
+      serviceName,
+    ]
+  );
+
+  const {
+    data: errorSamplesData = emptyErrorSamples,
+    status: errorSamplesFetchStatus,
+  } = useFetcher(
+    (callApmApi) => {
+      if (start && end) {
+        return callApmApi(
+          'GET /internal/apm/services/{serviceName}/errors/{groupId}/samples',
+          {
+            params: {
+              path: {
+                serviceName,
+                groupId,
+              },
+              query: {
+                environment,
+                kuery,
+                start,
+                end,
+              },
+            },
+          }
+        );
+      }
+    },
+    [environment, kuery, serviceName, start, end, groupId]
+  );
+
+  const { errorDistributionData, status: errorDistributionStatus } =
+    useErrorGroupDistributionFetcher({
+      serviceName,
+      groupId,
+      environment,
+      kuery,
+    });
+
+  useEffect(() => {
+    const selectedSample = errorSamplesData?.errorSampleIds.find(
+      (sample) => sample === errorId
+    );
+
+    if (errorSamplesFetchStatus === FETCH_STATUS.SUCCESS && !selectedSample) {
+      // selected sample was not found. select a new one:
+      const selectedErrorId = maybe(errorSamplesData?.errorSampleIds[0]);
+
+      history.replace({
+        ...history.location,
+        search: fromQuery({
+          ...omit(toQuery(history.location.search), ['errorId']),
+          errorId: selectedErrorId,
+        }),
+      });
+    }
+  }, [history, errorId, errorSamplesData, errorSamplesFetchStatus]);
+
+  // If there are 0 occurrences, show only charts w. empty message
+  const showDetails = errorSamplesData.occurrencesCount !== 0;
+
+  return (
+    <>
+      <EuiSpacer size={'s'} />
+
+      <ErrorGroupHeader
+        groupId={groupId}
+        occurrencesCount={errorSamplesData?.occurrencesCount}
+      />
+
+      <EuiSpacer size={'m'} />
+      <EuiFlexGroup>
+        <ChartPointerEventContextProvider>
+          <EuiFlexItem grow={3}>
+            <EuiPanel hasBorder={true}>
+              <ErrorDistribution
+                fetchStatus={errorDistributionStatus}
+                distribution={errorDistributionData}
+                title={i18n.translate(
+                  'xpack.apm.errorGroupDetails.occurrencesChartLabel',
+                  {
+                    defaultMessage: 'Error occurrences',
+                  }
+                )}
+                height={300}
+                tip={i18n.translate(
+                  'xpack.apm.serviceDetails.metrics.errorRateChart.tip',
+                  {
+                    defaultMessage: `Error rate is measured in transactions per minute.`,
+                  }
+                )}
+              />
+            </EuiPanel>
+          </EuiFlexItem>
+        </ChartPointerEventContextProvider>
+        <EuiFlexItem grow={2}>
+          <EuiPanel hasBorder={true}>
+            <MobileErrorTreemap
```

Comment:
This treemap should show the map related to the error group right? but it shows the overall errors  (The exact same as in the overview) as we don't filter by the error group

Reply:
fixed!